.redhat-ci.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -xeuo pipefail
+
+# https://bugzilla.redhat.com/show_bug.cgi?id=1318547#c7
+mount --make-rshared /
+
+systemctl start docker
+
+if [ -f /run/ostree-booted ]; then
+  ostree admin unlock
+fi
+
+# somewhat mimic the spec conditional
+source /etc/os-release
+if [ "$ID" == fedora ]; then
+  PYTHON=python3
+else
+  PYTHON=python
+fi
+
+DOCKER_RUN="docker run \
+              --privileged \
```

Comment:
This runs on dynamically provisioned hosts, so not too worried here. Though for good form we should.